crates/runtime/src/lib.rs

```diff
@@ -7,6 +7,7 @@ mod combine;
 mod container;
 mod derive;
 pub mod harness;
+mod iam_auth;
```

Comment:
`iam_auth` has enough going on that IMO it belongs as its own crate, which is _used_ by `runtime`. The singular `iam_auth.rs` is a bit chonky.